std/hash_map.zig

```diff
@@ -204,6 +204,10 @@ pub fn HashMap(comptime K: type, comptime V: type, comptime hash: fn (key: K) u3
             return hm.internalGet(key);
         }
 
+        pub fn getValue(hm: *const Self, key: K) ?V {
```

Comment:
i didn't put a doc comment on here, because it would pretty much be saying exactly what the method signature says. +1 for zig's expressive type system.